diff --git a/core/modules/content_translation/content_translation.module b/core/modules/content_translation/content_translation.module
index 36557cd..a5bb9f5f 100644
--- a/core/modules/content_translation/content_translation.module
+++ b/core/modules/content_translation/content_translation.module
@@ -299,6 +299,11 @@ function content_translation_form_alter(array &$form, FormStateInterface $form_s
     if (!$entity->isNew() && (!isset($translations[$form_langcode]) || count($translations) > 1)) {
       $langcode_key = $entity->getEntityType()->getKey('langcode');
       foreach ($entity->getFieldDefinitions() as $field_name => $definition) {
+        // Do not overwrite the #multilingual setting, if e.g. a module has
+        // already set it.
+        if (isset($form[$field_name]['#multilingual'])) {
+          continue;
+        }
         if (isset($form[$field_name]) && $field_name != $langcode_key) {
           $form[$field_name]['#multilingual'] = $definition->isTranslatable();
         }
