diff --git a/currency.services.yml b/currency.services.yml
index ad82ae3..36cb332 100644
--- a/currency.services.yml
+++ b/currency.services.yml
@@ -22,7 +22,7 @@ services:
     class: Commercie\Currency\Input
   currency.config_importer:
     class: Drupal\currency\ConfigImporter
-    arguments: ['@module_handler', '@event_dispatcher', '@config.typed', '@entity_type.manager']
+    arguments: ['@module_handler', '@config.typed', '@entity_type.manager']
   currency.form_helper:
     class: Drupal\currency\FormHelper
     arguments: ['@string_translation', '@entity_type.manager']
diff --git a/src/ConfigImporter.php b/src/ConfigImporter.php
index a01b01c..5cfb263 100644
--- a/src/ConfigImporter.php
+++ b/src/ConfigImporter.php
@@ -13,7 +13,6 @@
 use Drupal\Core\Config\TypedConfigManagerInterface;
 use Drupal\Core\Entity\EntityTypeManagerInterface;
 use Drupal\Core\Extension\ModuleHandlerInterface;
-use Symfony\Component\EventDispatcher\EventDispatcherInterface;
 
 /**
  * Provides a config importer.
@@ -52,13 +51,6 @@ class ConfigImporter implements ConfigImporterInterface {
   protected $currencyLocaleStorage;
 
   /**
-   * The event dispatcher.
-   *
-   * @var \Symfony\Component\EventDispatcher\EventDispatcherInterface
-   */
-  protected $eventDispatcher;
-
-  /**
    * The module handler.
    *
    * @var \Drupal\Core\Extension\ModuleHandlerInterface
@@ -77,18 +69,15 @@ class ConfigImporter implements ConfigImporterInterface {
    *
    * @param \Drupal\Core\Extension\ModuleHandlerInterface
    *   The module handler.
-   * @param \Symfony\Component\EventDispatcher\EventDispatcherInterface
-   *   The event dispatcher.
    * @param \Drupal\Core\Config\TypedConfigManagerInterface $typed_config_manager
    *   THe typed config manager.
    * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager
    *   The entity type manager.
    */
-  public function __construct(ModuleHandlerInterface $module_handler, EventDispatcherInterface $event_dispatcher, TypedConfigManagerInterface $typed_config_manager, EntityTypeManagerInterface $entity_type_manager) {
+  public function __construct(ModuleHandlerInterface $module_handler, TypedConfigManagerInterface $typed_config_manager, EntityTypeManagerInterface $entity_type_manager) {
     $this->currencyResourceRepository = new ResourceRepository();
     $this->currencyStorage = $entity_type_manager->getStorage('currency');
     $this->currencyLocaleStorage = $entity_type_manager->getStorage('currency_locale');
-    $this->eventDispatcher = $event_dispatcher;
     $this->moduleHandler = $module_handler;
     $this->typedConfigManager = $typed_config_manager;
   }
diff --git a/tests/src/Unit/ConfigImporterTest.php b/tests/src/Unit/ConfigImporterTest.php
index 76e1b9a..8e375e2 100644
--- a/tests/src/Unit/ConfigImporterTest.php
+++ b/tests/src/Unit/ConfigImporterTest.php
@@ -18,7 +18,6 @@
 use Drupal\currency\Entity\CurrencyInterface;
 use Drupal\currency\Entity\CurrencyLocaleInterface;
 use Drupal\Tests\UnitTestCase;
-use Symfony\Component\EventDispatcher\EventDispatcherInterface;
 
 /**
  * @coversDefaultClass \Drupal\currency\ConfigImporter
@@ -56,13 +55,6 @@ class ConfigImporterTest extends UnitTestCase {
   protected $entityTypeManager;
 
   /**
-   * The event dispatcher.
-   *
-   * @var \Symfony\Component\EventDispatcher\EventDispatcherInterface|\PHPUnit_Framework_MockObject_MockObject
-   */
-  protected $eventDispatcher;
-
-  /**
    * The module handler.
    *
    * @var \Drupal\Core\Extension\ModuleHandlerInterface|\PHPUnit_Framework_MockObject_MockObject
@@ -93,8 +85,6 @@ public function setUp() {
 
     $this->currencyLocaleStorage = $this->getMock(EntityStorageInterface::class);
 
-    $this->eventDispatcher = $this->getMock(EventDispatcherInterface::class);
-
     $this->moduleHandler = $this->getMock(ModuleHandlerInterface::class);
 
     $this->typedConfigManager = $this->getMock(TypedConfigManagerInterface::class);
@@ -108,14 +98,14 @@ public function setUp() {
       ->method('getStorage')
       ->willReturnMap($map);
 
-    $this->sut = new ConfigImporter($this->moduleHandler, $this->eventDispatcher, $this->typedConfigManager, $this->entityTypeManager);
+    $this->sut = new ConfigImporter($this->moduleHandler, $this->typedConfigManager, $this->entityTypeManager);
   }
 
   /**
    * @covers ::__construct
    */
   public function testConstruct() {
-    $this->sut = new ConfigImporter($this->moduleHandler, $this->eventDispatcher, $this->typedConfigManager, $this->entityTypeManager);
+    $this->sut = new ConfigImporter($this->moduleHandler, $this->typedConfigManager, $this->entityTypeManager);
   }
 
   /**
